packages/mobile/src/account/Settings.tsx

```diff
@@ -400,7 +404,9 @@ export class Account extends React.Component<Props, State> {
             {linkBankAccountEnabled && (
               <SettingsItemTextValue
                 title={t('linkBankAccountSettingsTitle')}
-                onPress={this.goToLinkBankAccount}
+                onPress={
+                  numberVerified ? this.goToLinkBankAccount : this.goToNumberNotConnectScreen
```

Comment:
I suggest that we only prompt the user to verify their phone number if they do not have an MTW address. It's possible that some users will have an MTW address but did not finish phone number verification. If that's the case, we don't care if they finish verifying their phone number or not; they have an MTW address and that's all we need.

Reply:
Its also possible that users have completed phone number verification and still won't have a MWT address https://valora-app.slack.com/archives/C02KBT0DAHJ/p1638554153012800?thread_ts=1638476912.011500&cid=C02KBT0DAHJ

Reply:
For now I think @cajubelt 's suggestion is a good one though.